code/modules/clothing/accessories/badges.dm

```diff
@@ -68,15 +68,12 @@
 		return 1
 
 /obj/item/clothing/accessory/badge/holo/attackby(var/obj/item/O as obj, var/mob/user as mob)
-	if(istype(O, /obj/item/weapon/card/id) || istype(O, /obj/item/device/pda))
+	if(istype(O, /obj/item/weapon/card/id) || istype(O, /obj/item/modular_computer))
```

Comment:
Unused check what conflict with idea of `proc/GetIdCard()`

Reply:
fixed